app/renderer/components/navigation/urlBarIcon.js

```diff
@@ -111,7 +111,9 @@ class UrlBarIcon extends React.Component {
 
   onDragStart (e) {
     dndData.setupDataTransferURL(e.dataTransfer, this.props.location, this.props.title)
-    dndData.setupDataTransferBraveData(e.dataTransfer, dragTypes.TAB, this.props.activeFrame)
+    dndData.setupDataTransferBraveData(e.dataTransfer, dragTypes.TAB, Immutable.fromJS({
```

Comment:
conversion to/from JS is a relatively expensive operation, why do we need to send an object instead of just the activeFrameKey value